### ISSUE TITLE
Need help on frequent enter & exit event

### ISSUE THREAD

**Issue by sudharsan1988** [mmazzarolo/react-native-beacons-android#25]
Hi Matteo,

I have got help from my friend to implement the background service to trigger in-app notification when user enter/exit in Beacon region but we are unable to control the frequency of notifications because of frequent enter & exit event.

Here is the updated code with Background service, it would be really great if you could help out.

https://www.dropbox.com/s/akhv69mflgmg1ru/react-native-beacons-android.zip?dl=0

Many thanks in advance!

**Comment by affibox** [comment]
Hi. Does you code detects beacons even when the app was killed or closed?

I need to be able to detect beacons like this.

**Comment by sudharsan1988** [comment]
Yes, but we can't control the frequency of in-app notification. Currently On Enter and On Exit events are happening very frequently.

**Comment by mmazzarolo** [comment]
@sudharsan1988 sorry for answering late but as I already stated [here](https://github.com/mmazzarolo/react-native-beacons-android/issues/29) I'm super busy and can't take a look at it at the moment :(
